File: roundcorners/src/main/java/com/kproduce/roundcorners/util/RoundMethodInterface.java
package com.kproduce.roundcorners.util;

/**
 * @author kuanggang
 */
public interface RoundMethodInterface {
    void setRadius(int radius);

    void setRadius(int radiusTopLeft, int radiusTopRight, int radiusBottomLeft, int radiusBottomRight);

    void setRadiusLeft(int radius);

    void setRadiusRight(int radius);

    void setRadiusTop(int radius);

    void setRadiusBottom(int radius);

    void setRadiusTopLeft(int radius);

    void setRadiusTopRight(int radius);

    void setRadiusBottomLeft(int radius);

    void setRadiusBottomRight(int radius);

    void setStrokeWidth(int width);

    void setStrokeColor(int color);

    void setStrokeWidthColor(int width, int color);
}

File: roundcorners/src/main/java/com/kproduce/roundcorners/util/RoundMethodInterface.kt
package com.kproduce.roundcorners.util

/**
 * @author kuanggang
 */
interface RoundMethodInterface {
    fun setRadius(radiusDp: Float)
    fun setRadius(radiusTopLeftDp: Float, radiusTopRightDp: Float, radiusBottomLeftDp: Float, radiusBottomRightDp: Float)
    fun setRadiusLeft(radiusDp: Float)
    fun setRadiusRight(radiusDp: Float)
    fun setRadiusTop(radiusDp: Float)
    fun setRadiusBottom(radiusDp: Float)
    fun setRadiusTopLeft(radiusDp: Float)
    fun setRadiusTopRight(radiusDp: Float)
    fun setRadiusBottomLeft(radiusDp: Float)
    fun setRadiusBottomRight(radiusDp: Float)
    fun setStrokeWidth(widthDp: Float)
    fun setStrokeColor(color: Int)
    fun setStrokeWidthColor(widthDp: Float, color: Int)
}